src/QueryBuilder/ResultViewer/DynamicTable/DynamicTable.js

```diff
@@ -1,9 +1,23 @@
 import React, { useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
 import PropTypes from 'prop-types';
 import css from './DynamicTable.css';
 
 const columnStyle = { width: '180px', minWidth: '180px' };
 
+function getCellValue(row, property) {
+  // typeof check to ensure we don't try to display null/undefined as a booleans
+  if (property.dataType.dataType === 'booleanType' && typeof row[property.property] === 'boolean') {
```

Comment:
I suggest to use isBoolean(value) from Lodash here. Any case it is in project.

Reply:
i don't feel this small check is deserving of pulling in a util from lodash. fixed other nit